internal/core/vulndb/exploitdb_service.go

```diff
@@ -53,13 +53,17 @@ func (s *exploitDBService) fetchCSV(ctx context.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "could not read csv")
 	}
-	exploits := make(map[int]models.Exploit, 0)
+	exploits := make(map[string]models.Exploit, 0)
 	for _, record := range records[1:] {
-		id, err := strconv.ParseInt(record[0], 10, 64)
-		if err != nil {
-			slog.Warn("could not parse exploit id", "id", id)
-			continue
-		}
+		/*
+			id, err := strconv.ParseInt(record[0], 10, 64)
+			if err != nil {
+				slog.Warn("could not parse exploit id", "id", id)
+				continue
+			}
+		*/
+		id := record[0]
```

Comment:
I would love to have both ids prefixed - this way, we know where the exploit comes from.

```suggestion
		id := fmt.Sprintf("exploitdb:%s", record[0])
```